client-reactjs/src/components/application/AssetDetailsDialog.js

```diff
@@ -48,8 +50,10 @@ function AssetDetailsDialog(props) {
     })
   }
 
+  //Mapping through all nodes and getting title of selected node
+  const assetName = props.nodes.filter((node) => props.selectedAsset ===  node[`${props.assetType}Id`] );
```

Comment:
Can the assetName be passed through props? Graph.js has title of the node available.

Reply:
Resolved